src/HPWHHeatSources.cc

```diff
@@ -60,12 +60,8 @@ HPWH::HeatSource& HPWH::HeatSource::operator=(const HeatSource& hSource)
     if (hSource.backupHeatSource != NULL || hSource.companionHeatSource != NULL ||
         hSource.followedByHeatSource != NULL)
     {
-        hpwh->simHasFailed = true;
-        if (hpwh->hpwhVerbosity >= VRB_reluctant)
-        {
-            hpwh->msg(
-                "HeatSources cannot be copied if they contain pointers to other HeatSources\n");
-        }
+        hpwh->send_error(
+            "HeatSources cannot be copied if they contain pointers to other HeatSources.");
```

Comment:
The HPWHHeatSource class should probably be its own Sender class have it's own courier to relay messages up through its parent HPWH.

Reply:
This message was marked resolved, but I don't see any changes.